scripts/check_imports_and_dependencies.py

```diff
@@ -47,7 +47,7 @@ def list_decorator(fn: Callable) -> Callable:
     """Wraps generator to return list."""
 
     @wraps(fn)
-    def wrapper(*args, **kwargs) -> List[Any]:
+    def wrapper(*args, **kwargs) -> List[Any]:  # type: ignore
```

Comment:
why types ignored?

Reply:
because it gave me error, probably merging with develop will fix that